src/main/java/hooks/Hooks.java

```diff
@@ -0,0 +1,37 @@
+package hooks;
+
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import dataProviders.ConfigFileReader;
+import managers.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+
+public class Hooks {
+    private ConfigFileReader fileReader;
+    private WebDriverManager webDriverManager;
+
+    private static WebDriver driver;
+
+    public Hooks() {}
```

Comment:
do you really need it?

Reply:
Yes, there is no need for the constructor here, thanks